scripts/docker-dos/dosUrlLocalizer.sc

```diff
@@ -55,16 +58,22 @@ case class DosObject(data_object: DosDataObject)
 
 case class GoogleServiceAccount(data: JsObject)
 
-case class MarthaResponse(dos: DosObject, googleServiceAccount: GoogleServiceAccount)
+case class MarthaResponse(dos: DosObject, googleServiceAccount: Option[GoogleServiceAccount])
 
 
 @main
 def dosUrlResolver(dosUrl: String, downloadLoc: String) : Unit = {
   val dosResloverObj = for {
     marthaUrl <- Uri.fromString(sys.env("MARTHA_URL")).toTry
+    requesterPaysProjectIdOption <- Try(sys.env.get("REQUESTER_PAYS_PROJECT_ID"))
```

Comment:
TOL: might as well constantly use the same "value-into-Try" logic as the previous line

Reply:
`.toTry` fails here as `sys.env.get("REQUESTER_PAYS_PROJECT_ID")` returns an Option[String]. Were you suggesting something else as part of "value-into-Try"?